app/controllers/phases_controller.rb

```diff
@@ -29,6 +29,13 @@ def create
     redirect_to project_lead_phases_path, notice: 'Phase was successfully created.'
   end
 
+  def update
+    if @phase.update(phase_params)
+      redirect_to project_lead_phases_url, notice: 'Phase Updated !'
+    else
+      render 'edit'
+    end
+  end
   def engineer
```

Comment:
spacing issue and name is not good, phases/engineer doesn't indicate anything

Reply:
Name Changed

Reply:
Space Issue Resolved